functions.php

```diff
@@ -137,7 +147,41 @@ function stock_scripts() {
 	}
 	h2.site-description {
 		color: {$secondary_color};
-	}";
+	}
+	body,
+	button {
+		color: {$text_color}
```

Comment:
Another nitpick: please add a semicolon to the end of these properties (for consistency and future editability).